helpers.php

```diff
@@ -1,22 +1,4 @@
 <?php
-/**
```

Comment:
It is, but in 5.4 this `retry` method was [moved into core](https://github.com/laravel/framework/blob/5.4/src/Illuminate/Support/helpers.php#L689).